admin-dev/themes/new-theme/js/pages/currency/form/components/CurrencyFormatter.vue

```diff
@@ -64,26 +81,47 @@
         required: true
       }
     },
-    components: {LanguageList, CurrencyModal},
+    components: {LanguageList, CurrencyModal, Modal},
     computed: {
       languagesCount() {
         return this.languages.length;
-      }
+      },
+      modalTitle() {
+        return this.$t('modal.confirm.title');
+      },
+      cancelLabel() {
+        return this.$t('modal.confirm.cancel');
+      },
+      confirmLabel() {
+        return this.$t('modal.confirm.apply');
+      },
+      modalBody() {
+        return this.$t('modal.confirm.body');
+      },
     }, methods: {
       closeModal() {
         this.selectedLanguage = null;
       },
       selectLanguage(language) {
         this.selectedLanguage = language;
       },
+      closeConfirmationModal() {
+        this.showConfirmationModal = false;
```

Comment:
You can simplify this by removing the boolean, and use `this.showConfirmationModal` as your condition, in the template simply use `v-if="confirmationLanguage !== null"`

Reply:
done